habitat-baselines/habitat_baselines/rl/ver/inference_worker.py

```diff
@@ -323,6 +323,7 @@ def step(self) -> Tuple[bool, List[Tuple[int, int]]]:
                 actions,
                 actions_log_probs,
                 next_recurrent_hidden_states,
+                policy_info,
```

Comment:
I am getting an error here, the `act` is returning `PolicyAction`, which cannot be unpacked. I am running this:

```
python -u habitat-baselines/habitat_baselines/run.py  \
--exp-config habitat-baselines/habitat_baselines/config/rearrange/rl_skill.yaml  \
--run-type train      \
benchmark/rearrange=place
```

Reply:
This should be fixed now.